image/Makefile

```diff
@@ -98,6 +100,11 @@ romfs: template
 	$(MAKE) -f Make.devices "MKDEV=echo >> romfs.devices"
 	mkromfs -d romfs.devices $(DESTDIR)
 
+# Create RAW filesystem
+
+raw: $(ELKS_DIR)/arch/i86/boot/Image
+	cp $(ELKS_DIR)/arch/i86/boot/Image $(TARGET_FILE)
```

Comment:
You should first create an empty disk image according to the selected format, then DD the `Image` after.